org.genivi.DLTViewer.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.genivi.DLTViewer
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: dlt-viewer
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=all
+  - --share=network
+  - --persist=.dlt
+  - --env=LD_LIBRARY_PATH=/app/usr/lib/x86_64-linux-gnu
```

Comment:
It's a weird thing to include. Are you sure it's actually needed? Note it's not really correct for aarch64.

Reply:
Yes. The application installs a library named qdlt at $PREFIX/$QT_INSTALL_LIBS [here](https://github.com/GENIVI/dlt-viewer/blob/master/qdlt/qdlt.pro#L35). And on x86_64 with org.kde.Sdk it has that path:
```
[abono@ryo flathub]$ flatpak run --command=sh org.kde.Sdk
[📦 org.kde.Sdk ~]$ qmake -query QT_INSTALL_LIBS
/usr/lib/x86_64-linux-gnu
```
The commit that introduced this change is [this one](https://github.com/GENIVI/dlt-viewer/commit/b51eebfb8ecb93f1771b9b862b922ad3573f683a) and I don't feel confident to change the install path without any regressions. Do you have any suggestions?

You are right though, it won't work for aarch64. I need to know the $QT_INSTALL_LIBS value on that architecture and add it as a workaround. Do you have access to an aarch64 machine?

Reply:
I've found the $QT_INSTALL_LIBS value on aarch64 systems by looking at the build logs of this pull request. I'm going to mark this as resolved. If you have suggestions on how to improve the overall install paths story please let me know (even by opening an issue on the upstream project).